packages/core/src/modules/attendance/attendee-service.ts

```diff
@@ -29,6 +30,10 @@ export interface AttendeeService {
   getByAttendanceId(attendanceId: string): Promise<AttendeeUser[]>
   getByAttendancePoolId(id: AttendancePoolId): Promise<AttendeeUser[]>
   updateAttended(attended: boolean, id: AttendeeId): Promise<Attendee>
+  handleQrCodeRegistration(
+    userId: UserId,
+    attendanceId: AttendanceId
+  ): Promise<{ attendee: Attendee; user: User; alreadyAttended: boolean }>
```

Comment:
I think it's easier to read if you move the return type into an interface/type

Reply:
I agree, it is now changed to QrCodeRegistrationAttendee type